CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Fix one_manager_agent_env pytest marker for System Tests ([#4763](https://github.com/wazuh/wazuh-qa/pull/4763)) \- (Tests)
```

Comment:
```suggestion
- Fix one_manager_agent_env pytest marker for System Tests ([#4782](https://github.com/wazuh/wazuh-qa/pull/4782)) \- (Tests)
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/4782/commits/0d439954a82568453dad5ce801a6ea4a839b6653